src/Uno.UI/UI/Xaml/Controls/ScrollContentPresenter/ScrollContentPresenter.wasm.cs

```diff
@@ -97,27 +93,29 @@ private void HandlePointerEvent(Input.PointerRoutedEventArgs e)
 			}
 		}
 
-		public ScrollMode HorizontalScrollMode
-		{
-			get => _horizontalScrollMode1;
-			set
-			{
-				_horizontalScrollMode1 = value;
-				SetClasses(HorizontalModeClasses, (int)value);
-			}
-		}
-
-		private static readonly string[] VerticalModeClasses = { "scrollmode-y-disabled", "scrollmode-y-enabled", "scrollmode-y-auto" };
-
-		public ScrollMode VerticalScrollMode
-		{
-			get => _verticalScrollMode1;
-			set
-			{
-				_verticalScrollMode1 = value;
-				SetClasses(VerticalModeClasses, (int)value);
-			}
-		}
+		//private static readonly string[] HorizontalModeClasses = { "scrollmode-x-disabled", "scrollmode-x-enabled", "scrollmode-x-auto" };
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5943777)